### PR TITLE
asset expectations

### PR DESCRIPTION
**Pull request by sryza**

### CODE REVIEW

File: python_modules/dagster-graphql/dagster_graphql/schema/logs/events.py [hunk 1]
```diff
@@ -332,6 +334,9 @@ def resolve_metadataEntries(self, _graphene_info):
 
         return _to_metadata_entries(self._metadata.metadata_entries)
 
+    def resolve_expectationResults(self, _graphene_info) -> Sequence[GrapheneExpectationResult]:
+        return "what goes here???"
```
Comment (sryza) [inline python_modules/dagster-graphql/dagster_graphql/schema/logs/events.py:338]: @clairelin135 are you able to help a graphql noob (me) with this? How do I turn `ExpectationResult` objects into `GrapheneExpectationResults`?

Comment (clairelin135) [inline python_modules/dagster-graphql/dagster_graphql/schema/logs/events.py:338]: expectation results are only a field on `AssetObservations` and not `AssetMaterializations` right? If that's the case, you'll need to change `GrapheneAssetObservation` instead:

```
class GrapheneObservationEvent(graphene.ObjectType, AssetEventMixin):
    class Meta:
        interfaces = (GrapheneMessageEvent, GrapheneStepEvent, GrapheneDisplayableEvent)
        name = "ObservationEvent"

    expectationResult = non_null_list(GrapheneExpectationResult)

    def __init__(self, event):
        self._observation = event.dagster_event.asset_observation_data.asset_observation
        super().__init__(**_construct_asset_event_metadata_params(event, self._observation))
        AssetEventMixin.__init__(
            self,
            event=event,
            metadata=self._observation,
        )

    def resolve_expectationResult(self, _graphene_info):
        return self._observation.expectation_results
 ```
 
 I think doing this should work!

Comment (sryza) [reply python_modules/dagster-graphql/dagster_graphql/schema/logs/events.py:338]: I think what's confusing me is that I expect the type of `self._observation.expectation_results` to be `ExpectationResult`, but I expect this to return a `GrapheneExpectationResult`.  What am I missing?

Comment (clairelin135) [reply python_modules/dagster-graphql/dagster_graphql/schema/logs/events.py:338]: This part I don't completely understand either, but graphene has a ["default resolver"](https://docs.graphene-python.org/en/latest/types/objecttypes/#defaultresolver) functionality that will fetch the attribute from the provided object given the field name. So I think in this case, `GrapheneExpectationResult` has fields like 'success', 'label', etc. that are basically mapped to `ExpectationResult.success` and `ExpectationResult.label`

Comment (sryza) [reply python_modules/dagster-graphql/dagster_graphql/schema/logs/events.py:338]: got it - that clears it up. thank you!

**Pull request closed without merging**